provisioning/README.md

```diff
@@ -0,0 +1,28 @@
+# console.redhat.com provisioning
+
+As a followup to our image-builder research, let's have a look at the provisioning service.
+
+This is not thorough at all since the service (and especially the API) is rough
+around the edges (April 2023). I'm assuming it will change substantially in
+the future.
+
+## Links
+
+- [API Docs](https://console.redhat.com/beta/docs/api/provisioning/v1)
+- [Real API examples](https://github.com/RHEnVision/provisioning-backend/tree/8898e972a48387b95b604e612e279ba155682c89/scripts/rest_examples)
+  - I needed these to know what to send to the API
+- More info in the issue: https://github.com/packit/research/issues/167
+
+## Notes
+
+- There is no UI: instances are launched in the Image Builder interface.
```

Comment:
What do you mean by UI in this case? (For who? Or on which side?) Can you please suggest what you would like to see?

Reply:
will clarify